resources/vip-template/VIP/Scene.xctemplate/UIViewController/___FILEBASENAME___Composer.swift

```diff
@@ -8,17 +8,14 @@
 
 import Foundation
 
-struct ___VARIABLE_sceneName___Composer {
+enum ___VARIABLE_sceneName___Composer {
 
-    static func compose(view: ___VARIABLE_sceneName___ViewProtocol) {
+    static func compose(view: ___VARIABLE_sceneName___ViewProtocol, router: ___VARIABLE_sceneName___RouterProtocol = ___VARIABLE_sceneName___Router()) {
```

Comment:
I'd like to have a session to discuss the router's role, there is still one more improvement that could be done here but I didn't feel this iteration was the best to introduce it